packages/composables/src/useUser/useUser.ts

```diff
@@ -180,8 +180,12 @@ export function useUser(): UseUserReturn {
         storefrontUrl: getStorefrontUrl(),
       },
     });
-    _user.value = data;
-    if (_user.value?.active) await refreshSessionContext();
+    // Update the user data in the context if the user is active and not using double opt-in registration set in the Shopware Admin
+    if (data.active && !data.doubleOptInRegistration) {
+      _user.value = data;
+      await refreshSessionContext();
```

Comment:
shouldn't we still refresh the context? User should not be in it

Reply:
hmm, I don't see the reason to refresh the context if the user is not "let in". Do you see any specific reason? Maybe I'm missing something.

Reply:
just to "make sure" - the action is pretty relevant from the system and user perspective. If there is some another "flag" or business logic allowing to use the account for some time etc.
this way we're sure that the view is in sync with the backend state

Reply:
done